vidarrtest-regression.json.in

```diff
@@ -350,7 +356,7 @@
             {
                 "metrics_calculate": "@CHECKOUT@/tests/calculate.sh",
                 "metrics_compare": "@CHECKOUT@/tests/compare.sh",
-                "output_metrics": "/.mounts/labs/gsi/testdata/manta/output_metrics/tumor-and-normal.metrics",
```

Comment:
I wouldn't change this. It provides info on the test setup (not sample identity) which may be useful while debugging